packages/pigeon/lib/objc_generator.dart

```diff
@@ -92,8 +92,16 @@ const Map<String, String> _propertyTypeForDartTypeMap = <String, String>{
   'Map': 'strong',
 };
 
-String? _objcTypeForDartType(String type) {
-  return _objcTypeForDartTypeMap[type];
+String? _objcTypeForPrimitiveDartType(String type) {
+  return _objcTypeForDartTypeMap.containsKey(type)
+      ? '${_objcTypeForDartTypeMap[type]} *'
+      : null;
+}
+
+String _objcTypeForDartType(String? classPrefix, String type) {
```

Comment:
Can you document this as well? I'm not sure what the class prefix is/is for.

Reply:
Done.